lang/en/local_pages.php

```diff
@@ -109,4 +109,15 @@
 $string['backtolist'] = "Back to pages list";
 $string['page'] = 'Page';
 $string['form'] = 'Form';
-$string['type'] = 'Type';
\ No newline at end of file
+$string['type'] = 'Type';
+$string['email'] = "Email";
+$string['text'] = "Text";
+$string['number'] = "Number";
+$string['checkbox'] = "Checkbox";
+$string['textarea'] = "Text Area";
+$string['select'] = "Select";
+$string['html'] = "HTML";
+$string['numeric'] = "Numeric";
+$string['validemail'] = 'Please supply a valid email address for {$a->email}';
+$string['pleasefillin'] = 'Please fill in {a->name}';
```

Comment:
Missing $ in {$a->name}

```suggestion
$string['pleasefillin'] = 'Please fill in {$a->name}';
```

Reply:
Nicely spotted - will commit update.